cdap-ui/app/cdap/components/AbstractWidget/SchemaEditor/RowButtons/FieldAttributes/FieldAttributesPopoverButton.tsx

```diff
@@ -98,10 +103,15 @@ function FieldPropertiesPopoverButton({
         }}
       >
         <fieldset className={classes.popoverContainer} disabled={disabled}>
-          <If condition={!isFlatRowTypeComplex(type)}>
-            <strong>No Attributes</strong>
-          </If>
-          <If condition={type === AvroSchemaTypesEnum.RECORD || type === AvroSchemaTypesEnum.ENUM}>
+          <If
+            condition={
+              (type === AvroSchemaTypesEnum.RECORD ||
```

Comment:
Could it be possible that we can add a comment explaining why we need such condition? Or could we possibly break it down into a function or const?